api_app/analyzers_manager/views.py

```diff
@@ -46,6 +48,16 @@ class AnalyzerListAPI(APIView):
     def get(self, request, *args, **kwargs):
         try:
             ac = self.serializer_class.read_and_verify_config()
+            custom_configs = get_custom_config_as_dict(
+                request.user, CustomConfig.Type.ANALYZER
+            )
+            for analyzer in ac.values():
```

Comment:
while your intentions are clear to me, it would be useful to add more context (comments) to help the reader to understand what is happening here

Reply:
also, we need a test for this logic too which proves that it behaves accordingly